### ISSUE TITLE
Annotation @RunOnVirtualThread doesn't work on class level for Websockets Next Client

### ISSUE THREAD

**Issue by andreiyusupau** [quarkusio/quarkus#45027]
### Describe the bug

@RunOnVirtualThread annotation doesn't work on class level for class annotated with @WebSocketClient.

### Expected behavior

I expect that @RunOnVirtualThread would apply to all methods inside websocket client (as it works for controllers https://quarkus.io/blog/virtual-threads-2/).

### How to Reproduce?

```java
//doesn't work, handlers are executed on worker threads
@RunOnVirtualThread
@WebSocketClient(path = "/ws")
public class ExampleWebsocketClient {

    //works fine
    @RunOnVirtualThread
    @OnTextMessage
    void onMessage(String message, WebSocketClientConnection connection) {
        
    }
}
```

### Output of `uname -a` or `ver`

Microsoft Windows [Version 10.0.22631.4460]

### Output of `java -version`

Amazon Corretto 21.0.5

### Quarkus version or git rev

3.17.2

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

gradle 8.10.2

**Comment by ozangunalp** [comment]
I see that it was fixed in #44925 both for server and client.

**Comment by mkouba** [comment]
> I see that it was fixed in [#44925](https://github.com/quarkusio/quarkus/pull/44925) both for server and client.

Yes, it was. Although it only tests the server.
